modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -59,6 +63,21 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  changeAddress(addr: string): this {
+    this._changeAddress = addr;
+    return this;
+  }
+
+  senderBalance(b: string): this {
```

Comment:
can we combine senderBalance and changeAddress into one method because these two value needs to be input together to be used correctly. Maybe TotalInputBalance is a better name than senderBalance because in theory inputs can be just a fraction of sender's total balance.